.gitignore

```diff
@@ -18,3 +18,6 @@ coverage_report/
 
 # Spec archives
 spec.tar.gz
+
+# Creds
+arango_live_server_config.json
```

Comment:
This should have all the info to log in using the python ArangoDB client. Username, password, url